packages/cli-kit/src/dependency.ts

```diff
@@ -213,6 +213,7 @@ export interface DependencyVersion {
 export async function addNPMDependenciesIfNeeded(
   dependencies: DependencyVersion[],
   options: AddNPMDependenciesIfNeededOptions,
+  force = false,
```

Comment:
I'd move this one to `AddNPMDependenciesIfNeededOptions` as one additional option.